### DISCUSSION TITLE
Deployment

### DISCUSSION THREAD

**Discussion by alwirihad** [ikhsan3adi/absensi-sekolah-qr-code#79] [Q&A]
permisi mas mau nanya, mohon maaf sebelumnya sekiranya saya kurang tau, saya mahasiswa ingin mencoba untuk mendeploy dengan vercel, itu bagaimana ya mas caranya? atau ada cara lain untuk deploynya?

**Comment by ikhsan3adi** [comment]
Belum pernah pake vercel mas🙏

**Comment by janglapuk** [comment]
Bisa tapi repot.

- Gak bisa migrate, kecuali sudah terhubung ke remote database menggunakan data dump hasil migrate lokal;
- Gak bisa generate QR code, kecuali aplikasi sudah support nyimpan data QR code ke object storage (S3) selain ke local storage (seperti `uploads/qrcode/*`);
- Di Vercel, direktori yang writable cuma `/tmp`, itupun terbatas hanya 512MB aja;
- Kalau mau maksa pake `/tmp`, aplikasi harus menyediakan _image proxy_ (seperti data hasil generate QR code), karena gak bisa baca langsung ke file yang tersimpan di `/tmp`;
- dan masalah lainnya yang belum ketahuan.

Kalau mau coba untuk eksperimen, _monggo_...
